scipy/sparse/sparsetools/csr.h

```diff
@@ -28,30 +28,37 @@
  *   Duplicate entries will be summed.
  *
  *   Complexity: Linear.  Specifically O(nnz(A) + min(n_row,n_col))
- * 
+ *
  */
 template <class I, class T>
-void csr_diagonal(const I n_row,
-                  const I n_col, 
-	              const I Ap[], 
-	              const I Aj[], 
+void csr_diagonal(const I k,
+                  const I n_row,
+                  const I n_col,
+	              const I Ap[],
+	              const I Aj[],
```

Comment:
if these lines are changed anyway, then maybe fix the inconsistent indentation as well?